### DISCUSSION TITLE
엔티티 생성 컨벤션 정의 : 테이블명, 컬럼 어노테이션 필수 정의

### DISCUSSION THREAD

**Discussion by jiwon83** [What-s-Your-Plan/wypl-backend#17] [Announcements]
테이블명 : `@Table(name = "엔티티명_tbl")`
컬럼명 : `@Column(name = "저장할 컬럼명")`

**Comment by KIMSEI1124** [comment]
해당 내용은 추후 문서로 작성한 뒤 닫도록 하겠습니다!

적어주셔서 감사합니다! 👍🏼
